src/TLMessage/TLMessage/ServerMessages/ChatPhoto.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace TelegramOSINT\TLMessage\TLMessage\ServerMessages;
+
+use TelegramOSINT\MTSerialization\AnonymousMessage;
+use TelegramOSINT\TLMessage\TLMessage\TLServerMessage;
+
+class ChatPhoto extends TLServerMessage
+{
+    /**
+     * @param AnonymousMessage $tlMessage
+     *
+     * @return bool
+     */
+    public static function isIt(AnonymousMessage $tlMessage)
+    {
+        return self::checkType($tlMessage, 'chatPhoto');
+    }
+
+    public function __construct(AnonymousMessage $tlMessage)
+    {
+        parent::__construct($tlMessage);
+    }
+
+    /**
+     * @return FileLocation
+     */
+    public function getBigPhoto()
+    {
+        $photo = $this->getTlMessage()->getNode('photo_big');
+
+        return new FileLocation($photo);
+    }
+
+    /**
+     * @return FileLocation
+     */
+    public function getSmallPhoto()
```

Comment:
These functions (`getSmallPhoto`, `getBigPhoto`) can form an interface, `PhotoInterface`, `UserProfilePhoto` would implement it.